doc/analytics.md

```diff
@@ -13,7 +17,7 @@ The toolkit provides an abstraction around analytics to make tracking pageviews,
 
 The minimum you need to use the analytics function is:
 
-1. Include the following files from /javascripts/govuk/analytics in your project:
+1. Include the following files from /javascripts/govuk/analytics_toolkit in your project:
```

Comment:
1. Include the following files from `/javascripts/analytics_toolkit` in your project: